app/screens/search/search.js

```diff
@@ -97,6 +95,7 @@ export default class Search extends PureComponent {
             cursorPosition: 0,
             value: props.initialValue,
             managedConfig: {},
+            recent: props.recent,
```

Comment:
moved `recent` to be component state controlled so we can have it in the list without waiting for the redux action to complete as is being batch dispatched with the results

Reply:
Why not change the action to not batch it with results instead of keeping two different lists of the recent results?

Reply:
it is a redux action and I don't want to cause regressions on the webapp

Reply:
Does the web app actually offer a list of recent search queries? I feel like that might only exist on mobile. Regardless, only 1/5 on that, so it's up to you